style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(Header.svg);
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 10px;
+  letter-spacing: 2px;
+  text-decoration: none;
+  background: none;
+  padding: 5px 5px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+}
+
+a:hover {
+  background-color: #6070ff;
+}
+
+a:enabled {
+  background-color: #fff;
+  color: #7f8cff;
+}
+
+a:disabled {
+  background-color: #5e6c84;
+}
+
+a:active {
+  background: #2230d2;
+}
+
+.toggleNav {
+  display: none;
+}
+
+img {
+  width: 10%;
+  height: auto;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 800;
+  line-height: 60px;
+}
```

Comment:
- [ ] Please ensure updating the headline heading to duplicate the **color** and the same exact **font family** from the Figma design system.

## Your Design
![image](https://user-images.githubusercontent.com/68030297/219698527-ce939c2f-eac4-481d-a799-5f1d29fdc74e.png)

## Figma Design
![image](https://user-images.githubusercontent.com/68030297/219698739-93647021-755b-40ed-8209-d69dff0489ea.png)

Check [Poppins](https://fonts.google.com/specimen/Poppins?query=popp) for the font family.